client/src/components/overview/style-selector/components/AllStyleThumbnails.jsx

```diff
@@ -8,8 +8,13 @@ function AllStyleThumbnails(props) {
   if(props.data) {
     comp = (
       <div className="all-styles-tn">
-      {props.data.photos.map((img,index) => (
-      <StyleThumbnail imageindex={index} key={`img-${index}`} image={img.thumbnail_url}/>
+      {props.data.map((img,index) => (
+      <StyleThumbnail
+        updateStyle={props.updateStyle}
+        imageindex={index} key={`img-${index}`}
```

Comment:
I like the prefix on the key property. Good way to stay organized. I always end up with a bunch of plain numbers that don't mean anything.